includes/shortcodes/class-wc-shortcode-products.php

```diff
@@ -504,10 +505,18 @@ protected function get_products_ids() {
 			} else {
 				$products = new WP_Query( $this->query_args );
 			}
-			$ids = wp_parse_id_list( $products->posts );
+
+			$ids = array(
```

Comment:
Maybe name this $return now it's not just ids? Nicpick.

Reply:
Or just `$data` shoulds better.